libs/api/domains/health-directorate/src/lib/models/dispensations.model.ts

```diff
@@ -44,9 +44,13 @@ export class Dispensation {
   @Field(() => Date, { nullable: true })
   lastDispensationDate?: Date
 
+  @Field(() => Date, { nullable: true })
+  nextDispensationDate?: Date
+
```

Comment:
Just be aware if the client has some funky GraphQL date handling this might resolve to something weird